### ISSUE TITLE
Gossamer is not expanding its connected peers

### ISSUE THREAD

**Issue by EclesioMeloJunior** [ChainSafe/gossamer#3700]
## Issue summary

- Since Gossamer starts syncing in the westend network with other substrate peers, it keeps connected with 10 peers (all bootnodes) and sometimes it loses 1 connection

- Gossamer should be able to discover more peers in the network

**Comment by q9f** [comment]
Yes, I noticed the same on Paseo where I only had 3 peers (the bootnodes).

**Comment by haikoschol** [comment]
Does this issue still exist? I've been running Westend nodes off `development` and another branch recently and never ran into it. The nodes usually end up with around 150 peers, which might even be a bit much. 🤔
